examples/google/example.go

```diff
@@ -94,53 +99,47 @@ func login(outputDir string, alg jwa.KeyAlgorithm, signGQ bool) error {
 	if err := json.Indent(&prettyJSON, pktJson, "", "  "); err != nil {
 		return err
 	}
-
 	fmt.Println(prettyJSON.String())
 
 	// Save our signer and pktoken by writing them to a file
-	return signer.WriteToFile()
+	saveLogin(outputDir, signer.(*ecdsa.PrivateKey), pktJson)
+
+	return nil
 }
 
-func googleSign(message string, outputDir string, alg jwa.KeyAlgorithm, signGq bool) error {
-	signer, err := pktoken.LoadFromFile(outputDir, alg.String(), signGq, nil)
+func sign(message string, outputDir string, alg jwa.KeyAlgorithm, signGq bool) error {
+	signer, pkt, err := loadLogin(outputDir)
 	if err != nil {
 		return fmt.Errorf("failed to load client state: %w", err)
 	}
 
-	msgHash := sha256.New()
-	_, err = msgHash.Write([]byte(message))
-	if err != nil {
-		return err
-	}
-	msgHashSum := msgHash.Sum(nil)
-
-	rawSigma, err := signer.Pksk.Sign(rand.Reader, msgHashSum, crypto.SHA256)
+	msgHashSum := sha3.Sum256([]byte(message))
+	rawSigma, err := signer.Sign(rand.Reader, msgHashSum[:], crypto.SHA256)
 	if err != nil {
 		return err
 	}
 
 	fmt.Println("Signed Message:", message)
 	fmt.Println("Praise Sigma:", base64.StdEncoding.EncodeToString(rawSigma))
-	fmt.Println("Hash:", hex.EncodeToString(msgHashSum))
+	fmt.Println("Hash:", hex.EncodeToString(msgHashSum[:]))
```

Comment:
Is the `[:]` needed? Does this fix a bug in my code?

Reply:
No, I'm just using a different function here which result in a `[32]byte` instead of a `[]byte` so I convert it to a byte slice by adding that

Reply:
Johnny pointed out the exact same [issue](https://github.com/openpubkey/openpubkey/pull/39#discussion_r1367172012), I explained my reasoning here and tagged you in the conversation. He originally resolved it, but I unresolved so you could see.